src/main/resources/config/liquibase/changelog/20240624091200_changelog.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+    <changeSet id="20240624091200" author="entholzer">
+        <createTable tableName="participation_vcs_access_token">
+            <column autoIncrement="true" name="id" type="bigint">
+                <constraints nullable="false" primaryKey="true"/>
+            </column>
+            <column name="user_id" type="bigint">
+                <constraints nullable="false"/>
+            </column>
+            <column name="participation_id" type="bigint">
+                <constraints nullable="false"/>
+            </column>
+            <column name="vcs_access_token" type="varchar(50)">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+        <addForeignKeyConstraint baseColumnNames="user_id"
+                                 baseTableName="participation_vcs_access_token"
+                                 constraintName="FK_participation_vcs_access_token_user"
+                                 referencedColumnNames="id"
+                                 referencedTableName="jhi_user"/>
+        <addForeignKeyConstraint baseColumnNames="participation_id"
+                                 baseTableName="participation_vcs_access_token"
+                                 constraintName="FK_participation_vcs_access_token_participation"
+                                 referencedColumnNames="id"
+                                 referencedTableName="participation"/>
+        <createIndex indexName="idx_vcs_access_token" tableName="participation_vcs_access_token">
+            <column name="vcs_access_token"/>
+        </createIndex>
+        <createIndex indexName="idx_participation_vcs_access_token_user_participation" tableName="participation_vcs_access_token">
+            <column name="user_id"/>
+            <column name="participation_id"/>
+        </createIndex>
```

Comment:
Is it currently possible to create multiple access tokens for the same user, participation pair in the data base? If so, maybe add a constraint checking that

Reply:
According to the model (`src/main/java/de/tum/in/www1/artemis/domain/participation/ParticipationVCSAccessToken.java`), the pair must be unique

Reply:
True, the class makes sure that it is unique, but now I also made sure that the participation-user pair is unique  in the migration :+1: